### ISSUE TITLE
compliance reporting nodes counts should NOT change when filtering by status

### ISSUE THREAD

**Issue by vjeffrey** [chef/automate#3177]
## Describe the bug
When calling the compliance/reporting/nodes API, counts are returned. These counts are used for the filter buttons in the UI.
It appears that these counts are being modified when a status filter is included in the request.
```
no filters: "total":5,"total_passed":0,"total_failed":5,"total_skipped":0,"total_waived":0}

with filter: 
curl -H "api-token: $ACCEPTANCE_TOKEN" -s https://a2-local-inplace-upgrade-acceptance.cd.chef.co/api/v0/compliance/reporting/nodes/search -d '{"filters": [{"type": "status", "values": ["passed"] } ]}'
{"nodes":[],"total":0,"total_passed":0,"total_failed":0,"total_skipped":0,"total_waived":0}
```

## To Reproduce
- call the compliance reporting nodes api. note counts.
- call the same api with a status filter. note difference in counts.

## Expected behavior
The counts returned with the API should be unaffected by a status filter. 
In other words, if i call the API with no filters, I should get a full list of nodes and the counts.
If I call the API with a status filter, I should get a list of nodes that have the same status as the requested one, but the counts should remain unchanged.
This will allow us to use backend filtering for the status filters on that page.

**Comment by vjeffrey** [comment]
related: https://github.com/chef/automate/issues/3178
